debezium-connector-binlog/src/main/java/io/debezium/connector/binlog/snapshot/lock/MinimalAtLeastOnceSnapshotLock.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.connector.binlog.snapshot.lock;
+
+import static io.debezium.connector.binlog.BinlogConnectorConfig.SnapshotLockingMode.MINIMAL_AT_LEAST_ONCE;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+
+import io.debezium.snapshot.spi.SnapshotLock;
+
+public class MinimalAtLeastOnceSnapshotLock implements SnapshotLock {
```

Comment:
So if we look at all the `SnapshotLock` modes the only ones that are not annotated with `@ConnectorSpecific` are those in `debezium-core`. @mfvitale can you remind me, do you think we'll need that here?

Reply:
The `@ConnectorSpecific` is used when there is the same snapshot mode in the core and in a specific connector. 

In this case `MinimalAtLeastOnceSnapshotLock` is just for binlog connector so it's fine.